tests/test_emulator.cpp

```diff
@@ -488,6 +488,18 @@ class EmulatorTest : public testing::Test {
         test_insn_mN_rN<N>(insn_name, tests, false);
     }
 
+    template <int N>
+    void test_bt(const char* insn_name,
+        const std::vector<test_alu_2op_t>& tests) {
```

Comment:
```suggestion
                 const std::vector<test_alu_2op_t>& tests) {
```

Reply:
I'm ready to merge this PR, but I don't know how to amend your original commit and apply this coding style suggestion... Let me know if you can do it easily :-)

Reply:
Sorry, my editor messed the function argument indentation (once again). I'll modify the settings so it doesn't happen another time. Fixed at f4884d4.